script/DeployAutoman.s.sol

```diff
@@ -45,11 +46,23 @@ contract DeployAutoman is Script {
         // Concatenate init code with encoded arguments
         bytes memory initCode = bytes.concat(type(UniV3Automan).creationCode, encodedArguments);
         bytes32 initCodeHash = keccak256(initCode);
-        console2.log("initCodeHash:");
+        console2.log("Automan initCodeHash:");
         console2.logBytes32(initCodeHash);
         // Compute the address of the contract to be deployed
         UniV3Automan automan = UniV3Automan(payable(create2deployer.computeAddress(salt, initCodeHash)));
 
+        if (address(automan).code.length != 0) {
```

Comment:
Does this work for Ethereum/Arbitrum mainnet? We updated our code once and the newly computed address will be different?

Reply:
Will hardcode the automan address for router deployment.